articles/connections/enterprise/okta.md

```diff
@@ -0,0 +1,212 @@
+* [Register your Application with Auth0](/getting-started/set-up-app). 
+  * Select an appropriate **Application Type**.
+  * Add an **Allowed Callback URL** of **`${account.callback}`**.
+  * Make sure your Application's **[Grant Types](/dashboard/guides/applications/update-grant-types)** include the appropriate flows.
+
+## Steps
+
+To connect your application to an OIDC Identity Provider, you must:
+
+1. [Register your app with Okta](#register-your-app-with-okta).
+2. [Create an enterprise connection in Auth0](#create-an-enterprise-connection-in-auth0).
+3. [Enable the enterprise connection for your Auth0 Application](#enable-the-enterprise-connection-for-your-auth0-application).
+4. [Test the connection](#test-the-connection).
+
+## Register your app with Okta
+
+To allow users to log in using Okta, you'll need to register your application. See [Register your app (Okta Documentation)](https://developer.okta.com/docs/guides/quickstart/website/register-app/) for full instructions.
+
+During registration you'll be prompted for **Sign-in redirect URIs**. Add your <dfn data-key="callback">callback URL</dfn> as a sign-in redirect URI:
+
+```
+https://${account.namespace}/login/callback
+```
+
+<%= include('../_find-auth0-domain-redirects.md') %>
+
+Once you finish registering your application with Okta, save the **Client ID** and **Client secret** to use in the next step.
+
+## Create an enterprise connection in Auth0
+
+Next, you will need to create and configure a OIDC Enterprise Connection in Auth0. Make sure you have the **Application (client) ID** and the **Client secret** generated when you set up your app in the OIDC provider.
+
+### Create an enterprise connection using the Dashboard
+
+::: warning
+To be configurable through the Auth0 Dashboard, the OpenID Connect (OIDC) Identity Provider (IdP) needs to support [OIDC Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html). Otherwise, you can configure the connection using the [Management API](#configure-the-connection-using-the-management-api).
+:::
+
+1. Navigate to [Auth0 Dashboard > Authentication > Enterprise](${manage_url}/#/connections/enterprise), locate **Open ID Connect**, and click its `+`.
+
+![Create Connection Type](/media/articles/connections/dashboard-connections-enterprise-list.png)
+
+2. Enter details for your connection, and select **Create**:
+
+| Field | Description |
+| ----- | ----------- |
+| **Connection name** | Logical identifier for your connection; it must be unique for your tenant. Once set, this name can't be changed. |
+| **Issuer URL** | URL where Auth0 can find the **OpenID Provider Configuration Document**, which should be available in the `/.well-known/openid-configuration` endpoint. You can enter the base URL or the full URL. You will see a green checkmark if it can be found at that location, a red mark if it cannot be found, or an error message if the file is found but the required information is not present in the configuration file. |
```

Comment:
We should give an example with how it would look with OKTA, eg. we don't need to say "which should be available" because we know where it is.